db/migrations/1657379568_create_courses_table.up.sql

```diff
@@ -2,9 +2,9 @@ BEGIN;
 
 CREATE TABLE courses
 (
-    id              bigserial       CONSTRAINT courses_pk PRIMARY KEY,
+    id              integer         PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```

Comment:
Keep bigserial here

Reply:
`id bigserial PRIMARY KEY GENERATED ALWAYS AS IDENTITY` seems to combine elements from two different approaches to generating unique identifiers in PostgreSQL.

We can use the data type `bigint`, which is designed to store very large integers. It can hold values from -9223372036854775808 to 9223372036854775807.

This choice is suitable for tables expected to grow beyond the capacity of the standard integer type, which is limited to a range of about -2 billion to +2 billion.

Reply:
go for bigint then

Reply:
https://github.com/sumelms/microservice-course/pull/19/commits/c04e66a4cce0b4bf28f7e13db39c5d4d1c41e173